### ISSUE TITLE
Add operations to handle enclosure urls

### ISSUE THREAD

**Issue by woorst** [newsboat/newsboat#235]
Much like there are operations like `open-in-browser` and `open-in-browser-and-mark-read` to open the url of an article it would be nice to have an operation like `open-media-in-browser` to open the enclosure url associated with an item so I can then do something like this in my config file:

    media-browser firefox
    macro i set media-browser "mpv %u"; open-media-in-browser ; set media-browser "firefox %u"

**Comment by Minoru** [comment]
Hi! I'm busy with ICFP Contest right now. I'll get back to you tomorrow when it's over. Thanks!

**Comment by Minoru** [comment]
Thank you for the patience! I think it would be better to add enclosure's URL to URLs dialog (invoked by `u` by default). The pros:
1. no new settings and commands (Newsboat has too many already, if you ask me);
2. will continue to work as expected when Newsboat adds support for multiple enclosures (right now it just picks last audio- or video-enclosure).

Cons:
1. instead of `,i`, you'll have to press `u,iq` (open URLs dialog, invoke your macro, quit the dialog).

How does that apply to your use-case, @woorst?

**Comment by woorst** [comment]
You're right, there are already so many commands. This seems like a good a option considering the upcoming support for multiple enclosures. 

Potential problem: The internal URL viewer does not work with an external HTML render.

I currently use `elinks` to render the HTML but the internal viewer seems to have gotten better so I'll switch back.

So, yeah. This solution will work.

**Comment by Minoru** [comment]
> Potential problem: The internal URL viewer does not work with an external HTML render.

That's a good point. But since enclosure URLs will be added by Newsboat itself, not renderer, that'll still work fine, albeit differently for depending on renderer used:
1. internal renderer: URLs dialog lists both enclosure URLs and links from the article;
2. external renderer: URLs dialog lists just enclosure URLs.

Theoretically we could run the article through internal renderer just to collect the links, and make the dialog work the same way for all users.
